cs-alibaba/license.template

```diff
@@ -0,0 +1,12 @@
+* (c) Copyright ${copyright.year} Micro Focus
```

Comment:
Need 2018

Reply:
@lrevnic : Do we need to edit the license.template file with the year 2018? 
As i have seen other files in cs-amazon/cs-azure, the template files doesn't contain the year, it is having the placeholder ${copyright.year}.